include/xrpl/protocol/Feature.h

```diff
@@ -372,6 +372,7 @@ extern uint256 const fixEnforceNFTokenTrustline;
 extern uint256 const fixInnerObjTemplate2;
 extern uint256 const featureInvariantsV1_1;
 extern uint256 const fixNFTokenPageLinks;
+extern uint256 const featureOnChainCredentials;
```

Comment:
Everywhere where "On-Chain Credentials" is used should be replaced with just "Credentials" - the on-chain is a given if you're in rippled.

Reply:
fixed